src/univalue/gen/gen.cpp

```diff
@@ -1,6 +1,6 @@
 // Copyright 2014 BitPay Inc.
 // Distributed under the MIT software license, see the accompanying
-// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+// file COPYING or https://opensource.org/licenses/mit-license.php.
```

Comment:
Univalue changes also need to go upstream.

Reply:
Thanks for pointing that out. Issued at https://github.com/jgarzik/univalue/issues/47 .